Directory.Build.props

```diff
@@ -1,18 +1,33 @@
 <Project>
+    <!-- Make F# support Central Package Management -->
     <PropertyGroup>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+        <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    </PropertyGroup>
+    <!-- Use lock files -->
+    <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true'">
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    </PropertyGroup>
+    <PropertyGroup>
+        <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+        <RestoreLockedMode Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</RestoreLockedMode>
+    </PropertyGroup>
+    <PropertyGroup>
+        <PackageProjectUrl>https://github.com/ionide/KeepAChangelog</PackageProjectUrl>
+        <Packagelicense>
+            https://github.com/ionide/KeepAChangelog/blob/master/LICENSE</Packagelicense>
+        <RepositoryUrl>https://github.com/ionide/KeepAChangelog</RepositoryUrl>
+        <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```

Comment:
We shouldn't use the License File here - we should instead use PackageLicenseExpression to provide a valid SPDX license expression. these expressions are much more usable by tooling than an arbitrary file.  This would mean that we would remove `PackageLicense` and the `None` item for the package license below.